content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -226,6 +252,35 @@ curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools
 .\PrepareNode.ps1 -KubernetesVersion {{< param "fullversion" >}}
 ```
 
+#### Add Windows Flannel and kube-proxy DaemonSets
```

Comment:
- Because Flannel is not part of Kubernetes, this doesn't comply with our content guide. Please read https://kubernetes.io/blog/2020/05/third-party-dual-sourced-content/ for context.
  Make sure to include the relevant third-party shortcode as part of this page rewrite.
- Setting up networking is not specific to containerd, so this advice seems to be in the wrong place.
- Overall, it might be better to link to the Flannel docs about how to set that up on a Windows node.
  - If those docs don't exist, make them (in the Flannel docs) and then hyperlink to the page you made.